tools/db_bench_tool.cc

```diff
@@ -2216,16 +2226,24 @@ class Stats {
   uint64_t bytes_;
   uint64_t last_op_finish_;
   uint64_t last_report_finish_;
+  CPUIOUtilizationTracker usage_tracker_;
+  AtomicRateTracker<size_t> req_drain_rate_;
   std::unordered_map<OperationType, std::shared_ptr<HistogramImpl>,
                      std::hash<unsigned char>>
       hist_;
   std::string message_;
   bool exclude_from_merge_;
   ReporterAgent* reporter_agent_;  // does not own
+  Options opts_;
   friend class CombinedStats;
 
  public:
-  Stats() : clock_(FLAGS_env->GetSystemClock().get()) { Start(-1); }
+  explicit Stats(const Options& option)
+      : clock_(FLAGS_env->GetSystemClock().get()),
+        usage_tracker_(option.rate_limiter),
+        opts_(option) {
+    Start(-1);
+  }
 
   void SetReporterAgent(ReporterAgent* reporter_agent) {
```

Comment:
Can you show the printing output of all these newly added reported things? I want to see why and whether it's worth it to include in production code.